_extras/checkout.md

```diff
@@ -102,44 +94,37 @@ Please note that:
     ~~~
     {: .source}
 
-5.  You only have to submit *one* change to move on to the next step,
-    not one per Carpentry.
+5.  We do *not* want exercises or other material that introduce new concepts:
+    our lessons are already full to bursting. If you are a Software Carpentry instructor and are
+    submitting an exercise, we will accept contributions to our [new Software Carpentry Python lesson]({{ site.swc_github}}/python-novice-gapminder/).
 
-6.  We do *not* want exercises or other material that introduce new concepts:
-    our lessons are already full to bursting.
-
-7.  If you are writing an exercise,
-    we strongly prefer contributions to the Data Carpentry lessons right now (since that material is much younger)
-    or to our fledgling "Python for complete novices" lesson.
-
-8.  If you would rather do a diagram, we are happy to take submissions to any of our lessons.
+6.  We are happy to accept contributions of new diagrams for any of our lessons.
     Please use SVG, since vector formats are easier to manipulate than raster formats.
 
-9.  We also appreciate improvements to our lessons' learning objectives.
-    In particular, we would like checked against the actual lesson content,
-    and would also like them labelled according to the levels of Bloom's Taxonomy.
+7.  We also appreciate improvements to our lessons' learning objectives.
+    In particular, we would like our current learning objectives to be checked against the actual lesson content.
```

Comment:
comma, and to be framed in terms of observables (e.g., "Learner will be able to do X") rather than intangibles (e.g., "Learner will appreciate X").